Sources/Moya/Plugins/NetworkLoggerPlugin.swift

```diff
@@ -18,10 +19,16 @@ public final class NetworkLoggerPlugin: PluginType {
     public let cURL: Bool
 
     /// Initializes a NetworkLoggerPlugin.
-    public init(verbose: Bool = false, cURL: Bool = false, output: ((_ separator: String, _ terminator: String, _ items: Any...) -> Void)? = nil, requestDataFormatter: ((Data) -> (String))? = nil, responseDataFormatter: ((Data) -> (Data))? = nil) {
+    public init(verbose: Bool = false,
+                cURL: Bool = false,
+                output: ((_ separator: String, _ terminator: String, _ items: Any...) -> Void)? = nil,
+                outputFormat: ((_ loggerId: String, _ date: String, _ identifier: String, _ message: String) -> String)? = nil,
```

Comment:
This looks good 👍 I'm just curious if another Moya contributor thinks that we should extract this to a `Formatter` object at this point.

Reply:
We could. We could also extract `output` as it also is not that clear. Maybe we could create two `typealiases` for these?

Reply:
@SD10 Are you thinking of a `Formatter` object that is responsible for both `output` and `outputFormat`?

@sunshinejr If both of you are keen on the separate `Formatter` object, then I guess there wouldn't be a need for the refactor you are proposing?

Reply:
Oh, if you are talking about both `output` & `ouputFormat` then I guess yeah 👍 Maybe the `Formatter` object name a bit more verbose, like `Output` as it is an output + output format.